specification/operationalinsights/resource-manager/Microsoft.OperationalInsights/stable/2015-03-20/OperationalInsights.json

```diff
@@ -1005,10 +1023,9 @@
     },
     "SavedSearchesListResult": {
       "properties": {
-        "metaData": {
+        "__metadata": {
           "x-ms-client-name": "Metadata",
-          "$ref": "#/definitions/SearchMetadata",
-          "description": "The metadata from search results."
```

Comment:
Is this a change in the behavior of the API, or was this property mis-documented in the first place?

Eitherway, I'd keep the line `"$ref": "#/definitions/SearchMetadata"` and not rename `metaData` to `__metadata`. For strongly typed languages, it could cause build breaks for applications. Even though both are failure cases, we need to be cognizant of how we're breaking our customers.

Reply:
That was a bug. The returned object has always an empty __metadata object and not metaData. So you prefer to leave it like that and not fix it?

Reply:
Ah, if the traffic is returning `__metadata` then you are right to update it to the correct name.

Reply:
That's the case. Can you sign off? Thanks!

Reply:
The other portion of this is that the type associated `__metadata` is stripped away. Please add that back.

Reply:
Added back. Will remove it in the next version.